docs/current_docs/ci/integrations/kubernetes.mdx

```diff
@@ -195,6 +195,13 @@ dagger query <<EOF
 EOF
 ```
 
+## Tested Configurations
+
+- Dagger's Helm chart is continuously tested [on K3s via a Dagger pipeline](https://github.com/dagger/dagger/blob/main/helm/.dagger/main.go#L41) in CI
+- Dagger has run CI jobs at significant scale on AWS EKS using the [approach outlined here](https://dagger.io/blog/argo-cd-kubernetes)
+- Dagger has tested [SUSE RKE2](https://docs.rke2.io/install/quickstart) and Rancher Desktop Kubernetes using our Helm chart in daemonset configuration. Recommend SLE Micro 6.1 with min 100GB SSD.
```

Comment:
```suggestion
- Dagger has tested [SUSE RKE2](https://docs.rke2.io/install/quickstart) and Rancher Desktop Kubernetes using our Helm chart in DaemonSet configuration. The recommended configuration is SUSE Linux Enterprise Micro 6.1 with a 100 GB SSD (minimum).
```

Reply:
@jpadams expanding the acronyms here

Reply:
Thanks! I amended this a bit.
Most descriptions actually use both expanded form and acronym:  `SUSE Linux Enterprise (SLE) Micro 6.1 `